formal-spec/candidateproperties.tex

```diff
@@ -0,0 +1,153 @@
+\section{Candidate Properties}
+
+\subsection{Global Properties}
+
```

Comment:
General remark: we should use the `property` environment for formal properties only. For the rest we can use a bulleted list for instance.